packages/web-twig/src/Resources/components/Toast/ToastBar.twig

```diff
@@ -15,7 +15,7 @@
 {%- set _rootDismissibleClassName = _isDismissible is same as(true) and _isTemplate is same as(false) ? _spiritClassPrefix ~ 'ToastBar--dismissible' : null -%}
 {%- set _boxClassName = _spiritClassPrefix ~ 'ToastBar__box' -%}
 {%- set _contentClassName = _spiritClassPrefix ~ 'ToastBar__content' -%}
-{%- set _messageClassName = _spiritClassPrefix ~ 'ToastBar__message' -%}
+{%- set _containerClassName = _spiritClassPrefix ~ 'ToastBar__container' -%}
```

Comment:
abc?

Reply:
🤷 we usually place root at top and then sort by ABC, but I will leave it up to you.